README.md

```diff
@@ -1,16 +1,21 @@
-# Title
+# Introduction to Active Record Associations
 
-## Learning Goals
+While you might be in the early days of writing code and learning how to program, 
+you have some  assets on your side: a positive attitude, a community you can ask 
+for help, and  a history of solving problems on your own. In this module, we're going 
+to give you  some language and a framework for the strategies that you should embrace 
+as you start solving problems in code.
 
--SWBAT 1
--SWBAT 2
+One of the most important skillsets for web development is the ability to think logically and to break down down problems into smaller, component pieces. This skillset is required when setting up your database. Data is the foundation of almost all web applications.
 
-## Introduction
+Active Record is the interface that Rails gives you to link the database to your application. It lets you structure your data models in a logical and nearly plain-English way. 
 
-## SWBAT 1
+We'll cover topics that will include answers to these questions:
 
-## SWBAT 2
+- What are the advantages of using ActiveRecord?
+- What are the features of Rake?
+- What are common methods accessible through Active Record associations?
+- How do you use association macros?
 
-## Conclusion
+Having a solid understanding of Active Record will likely make learning the rest of Rails seem eay. We’ll cover working with models, setting them up, and building associations between them. Over the next several lessons, we'll explain ActiveRecord and work with through a few labs that will help reinforce these concepts.
```

Comment:
Again rather than Rails, maybe just that Active Record replaces the need for a lot of the custom code we wrote in when creating ORMs, allowing us to create very simple but powerful Ruby classes that can persist data.